packages/manager/core/request-tagger/src/requestTagger.ts

```diff
@@ -0,0 +1,101 @@
+declare global {
+  interface Window {
+    ovhRequestTaggerNavigationId?: string;
+    ovhRequestTaggerRequestIndex?: number;
+  }
+}
+
+export enum Header {
+  NAVIGATION_ID = 'X-OVH-MANAGER-NAVIGATION-ID',
+  REQUEST_ID = 'X-OVH-MANAGER-REQUEST-ID',
+  PAGE = 'X-OVH-MANAGER-PAGE',
+  VERSION = 'X-OVH-MANAGER-VERSION',
+}
+
+export interface Headers {
+  [Header.NAVIGATION_ID]: string;
+  [Header.REQUEST_ID]: string;
+  [Header.PAGE]: string;
+  [Header.VERSION]?: string;
+}
+
+export interface HeadersOverrides {
```

Comment:
I think we can safely remove the `export` in all these, they're not used outside

Reply:
Used in tests, and could be helpful to share them ;)